src/crdt.rs

```diff
@@ -985,6 +1021,30 @@ mod tests {
         assert!(one && two);
     }
 
+    #[test]
+    fn purge_test() {
+        let me = ReplicatedData::new_leader(&"127.0.0.1:1234".parse().unwrap());
+        let mut crdt = Crdt::new(me.clone());
+        let nxt = ReplicatedData::new_leader(&"127.0.0.2:1234".parse().unwrap());
+        assert_ne!(me.id, nxt.id);
+        crdt.insert(&nxt);
+        let rv = crdt.gossip_request().unwrap();
+        assert_eq!(rv.0, nxt.gossip_addr);
+        let now = crdt.alive[&nxt.id];
+        let len = crdt.table.len() as u64;
+        crdt.purge(now);
+        let rv = crdt.gossip_request().unwrap();
+        assert_eq!(rv.0, nxt.gossip_addr);
```

Comment:
Inlining `rv` on all 4 assertions would make this test easier to read

Reply:
it makes really long lines, harder to read for some

Reply:
👍